site/content/documentation/programming/federation/index.md

```diff
@@ -445,6 +445,23 @@ FedX supports to use resolvable endpoints as federation members. These resolvabl
 
 Note that also hybrid combinations are possible.
 
+### Example 4: Local Federation with writable endpoint (NativeStore):
```

Comment:
This should be "Example 5"

As best practice I would suggest to also not restrict writability to a local federation with NativeStore (from the documentation perspective) => we should encourage users to rather move to repository manager based solutions and resolvable repositories (which of course then can be NativeStore). This is a more stable environment.

If you are OK, I will provide a commit on top